app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    result = split_integer(6, 2)
+    assert sum(result) == 6
```

Comment:
Code Style issue: As per the checklist, you should not create unnecessary variables for simple tests. You can directly use the function call in the assert statement. Example: `assert sum(split_integer(6, 2)) == 6`